### ISSUE TITLE
Maybe exposing some model hyper-parameter options to the user?

### ISSUE THREAD

**Issue by amznero** [shenweichen/DeepCTR#306]
Such as learning rate.
The default LR in estimator mode is fixed. 
 - Linear part: 0.005
 - DNN part: 0.01

in [deepctr/estimator/utils.py](https://github.com/shenweichen/DeepCTR/blob/master/deepctr/estimator/utils.py#L119-L121).

**Comment by zanshuxun** [comment]
Hi, LR in estimators can be modified outside by:
```python
...
from tensorflow.python.training import adagrad
from tensorflow.python.training import ftrl
...
    model = DeepFMEstimator(linear_feature_columns, dnn_feature_columns, task='binary', 
                            linear_optimizer=ftrl.FtrlOptimizer(0.05), dnn_optimizer=adagrad.AdagradOptimizer(0.1)
                            )

```

which is similar to [FAQ 2. Set learning rate](https://deepctr-doc.readthedocs.io/en/latest/FAQ.html#set-learning-rate-and-use-earlystopping)  in our doc except that optimizers are different:
```python
import deepctr
from tensorflow.python.keras.optimizers import Adam,Adagrad

model = deepctr.models.DeepFM(linear_feature_columns,dnn_feature_columns)
model.compile(Adagrad(0.1024),'binary_crossentropy',metrics=['binary_crossentropy'])
```

**Comment by amznero** [comment]
Got it!😅

Thanks for your reply, I'll try it in my code.

**Comment by amznero** [comment]
This issue was closed since the LR/Optimizer setting works fine.
